userspace/libsinsp/test/events_net.ut.cpp

```diff
@@ -381,9 +381,10 @@ TEST_F(sinsp_with_test_input, net_bind_listen_accept_ipv4) {
 	        increasing_ts(),
```

Comment:
I would remove the `BIND_E` event at line 372 to be sure we can correctly work with just the exit event